.github/workflows/ci.yml

```diff
@@ -249,10 +249,12 @@ jobs:
         uses: wearerequired/lint-action@v1
         with:
           stylelint: true
+          prettier_dir: common/src/main/webapp
```

Comment:
Isn't this file generated by sbt-lucuma now?

Reply:
Yes, that's why `build.sbt` changed in this PR